.ci/acceptance-tests/terraform-integration.yml

```diff
@@ -0,0 +1,22 @@
+platform: linux
+params:
+  # Params are set as environment variables when the run part is executed.
+  # Here we use (()) notation to indicate that we're using a credhub secret.
+  GOOGLE_JSON_ACCOUNT: ((terraform-integration-key))
+  GOOGLE_PROJECT: ((terraform-integration-project))
+  GOOGLE_ORG: ((terraform-integration-org))
+  GOOGLE_BILLING_ACCOUNT: ((terraform-integration-billing-account))
+  GOOGLE_PROJECT_NUMBER: ((terraform-integration-project-number))
+  TEST_DIR: ""
+  PROVIDER_NAME: ""
+  SHORT_NAME: ""
+  # TODO: GOOGLE_BILLING_ACCOUNT_2
+inputs:
+  - name: magic-modules-gcp
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: '1.11'
```

Comment:
How does the regular MM pipeline pick the golang version. Is it https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/containers/go-ruby/Dockerfile#L1 ? And is there a straight forward way of pulling out global configurations like this into a common place?